src/main/resources/init_db.sql

```diff
@@ -12,6 +12,8 @@ CREATE TABLE `drivers`  (
                             `name` varchar(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
                             `license_number` varchar(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
                             `is_deleted` bit(1) NOT NULL DEFAULT b'0',
+                            `login` VARCHAR(255) NOT NULL AFTER `is_deleted`,
+                            `password` VARCHAR(255) NOT NULL AFTER `login`;
```

Comment:
This is not compile, subtitute on this strings:
                            `login` varchar(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,
                            `password` varchar(255) CHARACTER SET utf8 COLLATE utf8_general_ci NOT NULL,

Reply:
and add 

DROP TABLE IF EXISTS `cars_drivers`;
DROP TABLE IF EXISTS `cars`;
DROP TABLE IF EXISTS `drivers`;
DROP TABLE IF EXISTS `manufacturers`;

in this file also after second string (and you can remove all redundant DROP before tables)